lib/Net/Async/Redis.pm

```diff
@@ -1096,6 +1096,46 @@ sub execute_command {
      ->retain;
 }
 
+=head2 RESP3 and RESP2 compatibility
+
+In RESP3 some of the responses are structured differently from RESP2 L<Net::Async::Redis> guarantees the same structure unless you have explicitly requested the new types
```

Comment:
This is a very wide line, which doesn't wrap in github (and probably most folks' editors). Please insert some linebreaks - 80 columns is traditional.